tesseract-server/src/app.rs

```diff
@@ -24,6 +24,10 @@ use crate::handlers::{
     metadata_all_handler,
     members_handler,
     members_default_handler,
+    logic_layer_geoadjacents_handler,
```

Comment:
As an overall theme, I think we should rename this endpoint to something a little broader than `geoadjacents` since it can support non-geo levels.

I am leaning towards something like `/relations` since that's broad enough to include everything, but I am open any ideas you might have.